app/src/main/java/com/mobiledevelopmentfoodapp/service/RestaurantService.kt

```diff
@@ -9,11 +9,11 @@ import kotlinx.coroutines.withContext
 import retrofit2.awaitResponse
 
 class RestaurantService {
-    suspend fun fetchFoods(): List<Food>?{
-        return withContext(Dispatchers.IO){
+    internal suspend fun fetchFoods(): List<Food>? {
```

Comment:
Added a visibility modifier to this function. Visibility modifier best practices can be found at https://kotlinlang.org/docs/visibility-modifiers.html